monkey/monkey_island/cc/testing/IslandTestCase.py

```diff
@@ -1,23 +1,41 @@
 import unittest
 
+import mongoengine
+
 import monkey_island.cc.environment.environment_singleton as env_singleton
 from monkey_island.cc.models import Monkey
 from monkey_island.cc.models.edge import Edge
 from monkey_island.cc.models.zero_trust.finding import Finding
 
 
 class IslandTestCase(unittest.TestCase):
+
+    def __init__(self, methodName):
+        # Make sure test is working with mongomock
+        if mongoengine.connection.get_connection().server_info()['sysInfo'] != 'Mock':
```

Comment:
If mongodb is not running and listening, would get_connection() work?

Reply:
yes